### ISSUE TITLE
Potential streams-dynamodb vs dynamodb signature issue 

### ISSUE THREAD

**Issue by codeasone** [cognitect-labs/aws-api#26]
Example code that fails: 

```
(comment
  (def ddbs-client (aws/client {:api :streams-dynamodb
                                :region :eu-west-1}))

  (aws/invoke ddbs-client {:op :DescribeStream
                           :request {:StreamArn example-arn}})
  ;; => {"__type" "com.amazon.coral.service#InvalidSignatureException",
  ;;     "message" "Credential should be scoped to correct service: 'dynamodb'. ",
  ;;     :cognitect.anomalies/category :cognitect.anomalies/incorrect}
  )
```

Environment: 

```
{:deps {org.clojure/clojure                {:mvn/version "1.10.0-RC2"}
        org.clojure/core.async             {:mvn/version "0.4.490"}
        com.cognitect.aws/api              {:mvn/version "0.8.155"}
        com.cognitect.aws/endpoints        {:mvn/version "1.1.11.462"}
        com.cognitect.aws/kinesis          {:mvn/version "668.2.357.0"}
        com.cognitect.aws/dynamodb         {:mvn/version "675.2.365.0"}
        com.cognitect.aws/streams-dynamodb {:mvn/version "669.2.364.0"}}}
```

__Note:__ I'm using credentials for default profile from my `.aws/credentials` which I have confirmed are working for other API usage e.g 

```
(comment
  (def ddb-client (aws/client {:api :dynamodb
                               :region :eu-west-1}))

  (aws/invoke ddb-client {:op :DescribeTable
                          :request {:TableName :secret-table-name}})
  ;; Succeeds
  )
```

**Comment by dchelimsky** [comment]
This is fixed, but not yet released (see [CHANGES.md](CHANGES.md)). I'll announce the release on the Clojure mailing list. I won't promise it today, but it should be in the next day or two.

**Comment by dchelimsky** [comment]
Released in 0.8.158.
